docs/aws-copilot.md

```diff
@@ -46,72 +82,64 @@ sso_registration_scopes = sso:account:access
 
 Running `aws sso login` will log you in via your browser.
 
-After this, AWS Copilot commands should run on your new profile:
+NOTE: If you have multiple AWS accounts, you can also use temporary credentials and export them. These can be found in the Command line or programmatic access bit under your account.
 
-```bash
-$ copilot app ls
-manage-childrens-vaccinations
-```
+After this, AWS Copilot will have access and visibility of the account and you can start using it.
 
-## Manual deployment
+## AWS Copilot - Starting from scratch
 
-Assuming you have an environment setup, go ahead and deploy:
+If setting it up the first time, or after tearing down the existent stack, follow the instructions below:
 
-```bash
-copilot svc deploy --env staging
-```
+**If you need to tear it down**
 
-## Opening a shell on the remote environment
+`copilot app delete`, and follow the instructions
 
-If you have the service up and running, you can connect to the first running
-container with this command:
+**Setting up a new Stack**
```

Comment:
Don't use `**Thing**` to denote a heading, use actual markdown headings and respect the hierarchy (level 1 heading at the top, and level 2/3/4 depending on section/subsection/etc)